besu/src/main/java/org/hyperledger/besu/BesuInfo.java

```diff
@@ -27,6 +29,21 @@ public final class BesuInfo {
   private static final String VERSION = BesuInfo.class.getPackage().getImplementationVersion();
   private static final String OS = PlatformDetector.getOS();
   private static final String VM = PlatformDetector.getVM();
+  private static final String COMMIT;
+
+  static {
+    if (VERSION == null) {
+      COMMIT = null;
```

Comment:
instead of NULL, would it be better to return empty String `""`?

Reply:
Also, is there a test that covers NULL commit? How would it be reported?

Reply:
Unfortunately, it's difficult to test the value of COMMIT because BesuInfo pulls data from the class' package which isn't typically set outside of a proper build.

As for null vs empty String, philosophically I'd rather it be null. The api doesn't really specify what should be supplied if a value isn't available.